ct_core/include/ct/core/Systems

```diff
@@ -15,14 +15,18 @@ Licensed under the BSD-2 license (see LICENSE file in main directory)
 #include "systems/continuous_time/linear/LTISystem.h"
 
 #include "systems/linearizer/DynamicsLinearizerNumDiff.h"
-#ifdef CPPADCG
+#ifdef CPPAD
 #include "systems/linearizer/DynamicsLinearizerAD.h"
```

Comment:
Unfortunately, DynamicsLinearizerADBase is not clean, there is a trait for a codgentype included in this base class.
Hence, CPPADCG is required to compile it at the current state.

Reply:
Okay, I see this has been fixed further down.